README.md

```diff
@@ -1852,6 +1852,20 @@ for char in 'pizza':
 
 <details>
 <summary>Write a function to return the sum of one or more numbers. The user will decide how many numbers to use.</summary><br><b>
+```
+First you ask the user for the amount of numbers that will be use. Use a while loop that runs until amount_of_numbers becomes 0 through subtracting amount_of_numbers by one each loop. In the while loop you want ask the user for a number which will be added a variable each time the loop runs.
+
+def returnsum():
```

Comment:
I know that this is nitpicking but let's achieve excellence and name it 
`sum_of_numbers` or `return_sum` for the glory of Pep 8?

Reply:
There are as well many other ways of solving this exercise; funny enough when we are learning programming, teachers usually shoves us while loops and use them to solve *every exercise*.
This builds a good logic thinking since most of the times, we have to think some mechanism to make the while work, in your case have a variable that decreases and another one that increases at every iteration. Usually you would use a while loop when you have an indefinite number of iterations and for loops in a definite number scenario.

```
def sum_of_numbers():
    amount_of_n = int(input("How many numbers? "))
    total_sum = 0

    for _ in range(amount_of_n):
        total_sum += int(input('Give number '))
    return total_sum
```
Feel free to use this snippet or any part of this comment if you want to fill a more complete answer, your answer is great nevertheless.

Reply:
Interesting implementation.
I actually aimed at providing numbers in the signature and using *args but I guess the blame on me with not providing clear description